tests/test_project_registrations.py

```diff
@@ -0,0 +1,248 @@
+import pytest
+from selenium.webdriver.common.action_chains import ActionChains
+
+import markers
+from api import osf_api
+from pages.project import RegistrationsPage
+from pages.registries import (
+    DraftRegistrationMetadataPage,
+    DraftRegistrationReviewPage,
+)
+from pages.support import SupportPage
+
+
+@pytest.fixture()
+def registrations_page(driver, default_project):
+    """Fixture that uses a temporary project created in the testing environments and
+    then navigates to the Registrations page for that project.
+    """
+    registrations_page = RegistrationsPage(driver, guid=default_project.id)
+    registrations_page.goto()
+    return registrations_page
+
+
+@pytest.fixture()
+def registrations_page_with_draft(session, registrations_page):
+    """This fixture uses the registrations_page fixture above and adds a draft
+    registration to the temporary project.  NOTE: Since we are creating the draft
+    registration from a temporary project that gets automatically deleted when we are
+    done with it, the draft registration as a child of the project will also get
+    deleted.
+    """
+
+    # First get the list of allowed registration schemas for OSF in a name and id pair
+    # list. Then loop through the list to pull out just the id for the Open-Ended
+    # Registration schema. We'll need this schema id to create the draft.
+    schema_list = osf_api.get_registration_schemas_for_provider(
+        provider_id='osf', data_type='name_id'
+    )
+    for schema in schema_list:
+        if schema[0] == 'Open-Ended Registration':
+            schema_id = schema[1]
```

Comment:
Unless you're expecting this `if` to succeed multiple times during the loop, add a `break` statement after this to exit the loop.

Reply:
Oops again. Fixed!